Fix build

error[E0658]: arbitrary expressions in key-value attributes are unstable
 --> /mnt/portagetmp/portage/games-rpg/rpg-cli-1.0.0/work/cargo_home/gentoo/clap-3.0.0-beta.4/src/lib.rs:8:10
  |
8 | #![doc = include_str!("../README.md")]
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: see issue #78835 <https://github.com/rust-lang/rust/issues/78835> for more information

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -5,7 +5,6 @@
 
 #![doc(html_logo_url = "https://clap.rs/images/media/clap.png")]
 #![doc(html_root_url = "https://docs.rs/clap/3.0.0-beta.4")]
-#![doc = include_str!("../README.md")]
 //! <https://github.com/clap-rs/clap>
 #![crate_type = "lib"]
 #![deny(
